kerastuner/tuners/bayesian.py

```diff
@@ -73,20 +74,34 @@ def __init__(self,
         self._tried_so_far = set()
         self._max_collisions = 20
         self.gpr = gaussian_process.GaussianProcessRegressor(
+            kernel=gaussian_process.kernels.Matern(),
+            n_restarts_optimizer=20,
+            normalize_y=True,
             alpha=self.alpha)
 
     def _populate_space(self, trial_id):
         # Generate enough samples before training Gaussian process.
         completed_trials = [t for t in self.trials.values()
                             if t.status == 'COMPLETED']
-        if len(self.trials) < self.num_initial_points or len(completed_trials) < 2:
+
+        # Use 3 times the dimensionality of the space as the default number of
+        # random points.
+        dimensions = len(self.hyperparameters.space)
+        num_initial_points = self.num_initial_points or 3 * dimensions
```

Comment:
This should be mentioned in the docstring

Reply:
Done